src/indexCors.ts

```diff
@@ -31,8 +31,12 @@ app.all('*', async (req: Request, res: Response) => {
     headers: {...headers,
       'X-Forwarded-For': ipString,
       'Forwarded': `for=${ipString}`
-    } as { [key: string]: string },
-    body: JSON.stringify(body)
+    } as { [key: string]: string }
```

Comment:
I'm curious, why is `headers` not a compliant type for `RequestInit`?

Reply:
looks like they are. I think this was a gpt error that I mistakenly didn't correct. will fix